apps/emqx_management/src/emqx_mgmt_api_clients.erl

```diff
@@ -855,6 +838,8 @@ ms(created_at, X) ->
 %%--------------------------------------------------------------------
 %% Match funcs
 
+fuzzy_filter_fun([]) ->
+    undefined;
```

Comment:
Just confirming: do we have some test coverage for this clause? (and similar ones in the other modules)

Reply:
Recently, I have added some test cases to all the new code added. thanks ❤️